orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -71,6 +73,14 @@ class MonitorKatoTask implements RetryableTask, CloudProviderAware {
 
   long getTimeout() { 3600000L }
 
+  @Override
+  long getDynamicBackoffPeriod(Stage stage, Duration taskDuration) {
+    if ((stage.context."kato.task.lastStatus" as ExecutionStatus) == ExecutionStatus.TERMINAL) {
+      return Math.max(backoffPeriod, TimeUnit.MINUTES.toMillis(2))
+    }
+    return backoffPeriod
```

Comment:
this is extra defensive programming Mark talking: do you think worth wrapping in `try/catch` so we never fail during task planning?
e.g.
```java
try {
    if ((stage.context."kato.task.lastStatus" as ExecutionStatus) == ExecutionStatus.TERMINAL) {
      return Math.max(backoffPeriod, TimeUnit.MINUTES.toMillis(2))
    }
} catch(e) {
  log(e)
}

return backoffPeriod
```

Reply:
I don't think that'll be necessary (famous last words).  The `if` statement should be safe.  If `stage.context."kato.task.lastStatus"` is null that'll just return false.